Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -1150,8 +1153,87 @@ final class HTTPClientTests: BaseHTTPClientTests {
             .toNot(contain(unexpectedDNSError.description))
     }
 
+    func testResponseFromServerUpdatesRequestDate() throws {
+        let path: HTTPRequest.Path = .mockPath
+        let mockedResponse = BodyWithDate(data: "test", requestDate: Date().addingTimeInterval(-3000000))
+        let encodedResponse = try mockedResponse.asJSONEncodedData()
+        let requestDate = Date().addingTimeInterval(-100000)
```

Comment:
What do these numbers mean? We could also hardcode a date for this and the `mockedResponse` request date to make it more deterministic.

Reply:
It's just a simpler way to make a date that's not "now" to avoid false negatives.